src/interface_adapter/Authentication/AuthenticationPresenter.java

```diff
@@ -36,6 +36,7 @@ public AuthenticationPresenter(ViewManagerModel viewManagerModel,
     @Override
     public void prepareSuccessView(AuthenticationOutputData authenticationOutputData) {
         DashboardState dashboardState = dashboardViewModel.getState();
+        dashboardState.setRightPanelView("dashboard");
```

Comment:
Added this so that when the user enters the dashbaord screen, it will never display any other view in the right panel such as the create account view or the view for other use cases. This ensures that the rightPanelView is always dashboard.